Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -263,15 +258,20 @@ class TabTrayController: UIViewController, UITabBarDelegate, UITableViewDelegate
 
 extension TabTrayController: Transitionable {
     private func getTransitionCell(options: TransitionOptions, browser: Browser?) -> CustomCell {
+        var transitionCell: CustomCell
         if let cell = options.moving as? CustomCell {
-            cell.tab = browser
-            return cell
+            transitionCell = cell
         } else {
-            let cell = CustomCell(style: UITableViewCellStyle.Default, reuseIdentifier: "id")
-            cell.tab = browser
-            options.moving = cell
-            return cell
+            transitionCell = CustomCell(style: UITableViewCellStyle.Default, reuseIdentifier: "id")
+            options.moving = transitionCell
         }
+
+        if let browser = browser {
+            transitionCell.background.image = screenshotHelper.takeScreenshot(browser, aspectRatio: nil, quality: 1)
```

Comment:
If there's no browser passed here, we'll just use whatever image was already in this cell. Is that intended?